src/cli.js

```diff
@@ -13,10 +13,7 @@ const generate = lazyMethodRequire("./commands/generate");
 const destroy = lazyMethodRequire("./commands/destroy");
 const dockerize = lazyMethodRequire("./commands/dockerize");
 
-const logger = require("./lib/logger");
```

Comment:
Looks like your branch might be killing the work Or did to clean up our usage of `chalk` into a logging helper. Probably just a merge conflict but we should use the `logger` stuff Or added